sailthru/sailthru_content/services/tests/catalog_api_test_mixins.py

```diff
@@ -45,3 +53,12 @@ def prepare_get_programs(self):
             content_type='application/json',
             match_querystring=False
         )
+
+    def prepare_searchable_courses(self):
+        responses.add(
+            responses.GET, self.api_url_root + '/search/all/facets/',
```

Comment:
I suspect that if you switch to the course run search endpoint: /api/v1/search/course_runs/ you will avoid the need to use the dictionary and the need to throw away programs entirely